src/server/graphql/subscriptions/newAuthToken.js

```diff
@@ -0,0 +1,15 @@
+import {GraphQLString} from 'graphql';
+import makeSubscribeIter from 'server/graphql/makeSubscribeIter';
+import {getUserId} from 'server/utils/authorization';
+import {NEW_AUTH_TOKEN} from 'universal/utils/constants';
+
+export default {
+  type: GraphQLString,
+  subscribe: (source, args, {authToken}) => {
+    const userId = getUserId(authToken);
+
+    // RESOLUTION
+    const channelName = `${NEW_AUTH_TOKEN}.${userId}`;
+    return makeSubscribeIter(channelName);
```

Comment:
in a future PR I'll extract the auth token from notifications & send a separate message that can can completely break out of the graphql data flow.